src/Native/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -11,6 +11,12 @@ add_compile_options(-Wno-cast-align)
 
 add_definitions(-DPIC=1)
 
+if(CMAKE_STATIC_LIB_LINK)
+   message("Build OpenSSL Static Link")
+   set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
+   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lcrypto -lz -ldl  -static-libgcc -static-libstdc++ -static")
```

Comment:
@bartonjs, I did a check again and you are right. I tested and remove completely the line that set CMAKE_EXE_LINKER_FLAGS. This is not required in our case.